docs/installation/sources/sources_linux.rst

```diff
@@ -0,0 +1,210 @@
+.. _linux_sources:
+
+Linux installation from sources
+===============================
+
+To install *eProsima Fast DDS* from sources, you first need to meet the required dependencies
+(see requirements_linux_sources_)
+and then choose whether to follow either the colcon_ (see colcon colcon_installation_linux_) or the CMake_
+(see cmake_installation_linux_) installation instructions.
+
+.. _requirements_linux_sources:
+
+Requirements
+------------
+
+*eProsima Fast DDS* requires the following dependencies when building from sources in a Linux environment.
+
+Asio and TinyXML2 libraries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can install these libraries using the package manager of your Linux distribution.
+For example, on Ubuntu you can install them with the command:
+
+.. code-block:: bash
+
+    sudo apt install libasio-dev libtinyxml2-dev
+
+Python
+^^^^^^
+
+Download and install the Python_ version that better fits your requirements.
+
+CMake, g++, pip, wget and git
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can install CMake_, g++, pip_, wget_ and git_ using the package manager of your Linux distribution.
+For example, on Ubuntu you can install them with the command:
+
+.. code-block:: bash
+
+    sudo apt install cmake g++ pip wget git
+
+OpenSSL
+^^^^^^^
+
+You can install OpenSSL_ using the package manager of your Linux distribution.
+For example, on Ubuntu you can install it with the command:
+
+.. code-block:: bash
+
+   sudo apt install libssl-dev
+
+Gtest
+^^^^^
+
+.. note::
+
+    By default, *eProsima Fast DDS* does not compile tests.
+    You can activate them by adding the :code:`-DPERFORMANCE_TESTS=ON` flag when calling colcon_ or CMake_
```

Comment:
There are many more tests than performance. I'd simply refer to the cmake options section instead of listing all the options that enable tests.

Reply:
ok.